pkgs/development/python-modules/powerline-mem-segment/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib,
+  buildPythonPackage,
+  fetchPypi,
+  psutil
+}:
+
+buildPythonPackage rec {
+  pname = "powerline-mem-segment";
+  version = "2.4.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0507zw7g449zk7dcq56adcdp71inbqfdmp6y5yk4x4j7kkp6pii9";
+  };
+
+  buildInputs = [ psutil ];
```

Comment:
Is that required at runtime? If so it needs to go to propagatedBuildInputs.

Reply:
Good catch, fixed